### ISSUE TITLE
As a member of the public who just went to a museum, I want to find zoomable display of artwork I just saw to show my family

### ISSUE THREAD

**Issue by zimeon** [IIIF/iiif-stories#30]
As a member of the public who just went to a museum, I want to find zoomable display of artwork I just saw to show my family

From 2015-12-11 IIIF Ghent Friday Unconference

**Comment by zimeon** [comment]
Implies ability to search based on information one might remember about the item just seen, perhaps title, artist, museum holding item

**Comment by saracarl** [comment]
Realistically " a member of the public" is going to use Google for this, or a search bar on the institution's (content management?) website.  Is IIIF really the solution here?  Or if it is, I think the discussion is more about the interaction of IIIF and the commodity search engines.

**Comment by beaudet** [comment]
I couldn't agree more.  I don't think the IIIF presentation API is currently a solution well suited for sharing significant quantities of cultural object metadata and I've heard it stated on more than one occasion that IIIF in general is deliberately NOT focused on cultural object discovery right now (if ever).  I would be careful reading too much into the fact that published manifests are now being used to find cultural object metadata and links to more comprehensive metadata service URLs embedded in those manifests. That said, I think the community would benefit greatly from a purposeful effort to developed standards that would help to bridge the gap between current practices and the holy grail of fully open and linked semantic web exposed via distributed SPARQL queries.

**Comment by azaroth42** [comment]
I agree that the *museum* community would benefit from practical effort similar to IIIF in nature for useful, semantic descriptions of objects. IIIF is not the right vehicle for that effort.

**Comment by beaudet** [comment]
Perhaps not, although I can see IIIF consciously not excluding the potential for compatibility between the IIIF discovery API and more domain-specific standards that would be detailed outside of the IIIF community.

**Comment by azaroth42** [comment]
Yep, `seeAlso` and `profile` :)

**Comment by beaudet** [comment]
Does the spec allow seeAlso + profile in collections or would it require a dummy manifest for any object without a IIIF resource?

**Comment by azaroth42** [comment]
Any resource can have a `seeAlso`, from Collections and Manifests, to Annotations, Images, Video, whatever you want.

**Comment by mattmcgrattan** [comment]
IIIF Discovery Call: 12-July-2017 Agreed to be out of scope for the discovery working group.

**Comment by beaudet** [comment]
Acknowledged and agree that it's out of scope, but I think the working group should be careful not to propose standards that would explicitly exclude the use of seeAlso and profile in data sets used by the Discovery API.
